README.md

```diff
@@ -187,18 +103,20 @@ Then report its outcome and end time:
     --run_step_end_time "Now"
 ```
 
-Don't forget to report the end of the build itself!
+Don't forget to report the end of the build itself, and information about the artifact generated!
 
 ```sh
 ./faros_event.sh CI -k "<faros_api_key>" \
     --run "<run_source>://<run_organization>/<run_pipeline>/<run_id>" \
     --run_status "Success" \
     --run_end_time "Now"
+    --artifact "<artifact_source>://<artifact_organization>/<artifact_repository>/<artifact_id>" 
```

Comment:
`--artifact` is dependent on `--commit` being provided in a partial CI event. It can't be split across partial events unfortunately.